package.json

```diff
@@ -45,8 +45,8 @@
         "csurf": "^1.11.0",
         "db.js": "^0.15.0",
         "debug": "^4.3.4",
-        "enketo-core": "^6.1.7",
-        "enketo-transformer": "2.1.7",
+        "enketo-core": "7.0.0",
```

Comment:
I think this should still be `^` right?

Reply:
I don't know? 😄 I changed it to be consistent with Transformer and because we need to update the changelog when we bump those anyway so it kind of makes sense to me to make both version increases explicit. I don't feel strongly about it.

Reply:
Fair enough, I was just erring on the side of doing things the same way they're already done unless there's a reason to change. Now that I think of it, `^` would only make a difference when consuming Express as a library anyway.